Dockerfile

```
@@ -34,13 +39,12 @@ RUN ["apt", "install", "-y", "ca-certificates"]
 WORKDIR /
 
 ENV HELM_PLUGINS /opt/helm-plugins
+RUN useradd  -r -u 499 nonroot
+RUN getent group nonroot || groupadd -o -g 499 nonroot
 
```

Let us make those exec-style calls. Also, I don't think we need this `groupadd`.
```suggestion
RUN ["useradd", "-r", "-u", "499", "nonroot"]
```

---

We're actually not using the `USER` directive afterwards, are we?

---

No, so it appears rest of the command is done a default user, but `manager.yaml` uses following security context for runtime:
```
runAsNonRoot: true
runAsUser: 499
runAsGroup: 499
```